.github/workflows/wheel_tests_and_release.yml

```
@@ -0,0 +1,204 @@
+name: Build Wheels and Release
+on:
+  push:
+    branches:
+      - cibuildwheel
+    tags:
+      - 'v*'
+      - 'buildwheels*'
+env:
+  CIBW_BUILD_VERBOSITY: 2
+  # Below is run within docker containers
+  # Only relevant for Linux
+  CIBW_BEFORE_ALL_LINUX: find .; bash bin/ci/linux_install_libfftw.sh
+  CIBW_TEST_REQUIRES: scipy dask
+  CIBW_TEST_COMMAND: >
+    python -c "import pyfftw; import numpy as np; from pyfftw.interfaces import numpy_fft as fft; np.testing.assert_allclose(fft.fft(np.zeros(8)), np.zeros(8));"
+
+jobs:
+  build_linux_wheels:
+    name: Build python ${{ matrix.cibw_python }} wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04]
+        cibw_python: ["cp37-*", "cp38-*", "cp39-*", "cp310-*"]
+        cibw_manylinux: [manylinux2014]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.9'
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+        with:
+          platforms: arm64
+
+      - name: Install cibuildwheel
+        run: |
+          python -m pip install cibuildwheel
+
+      - name: Build the wheel
+        run: |
+          python -m cibuildwheel --output-dir dist
+        env:
+          CIBW_BUILD: ${{ matrix.cibw_python }}
```

```suggestion
          CIBW_SKIP: "*-musllinux_*"
          # skip aarch64 tests which often exceed the 6 hour time limit
          CIBW_TEST_SKIP: "*_aarch64"
          CIBW_BUILD: ${{ matrix.cibw_python }}
```

I think I did have problems for `aarch64` and `musllinux` builds and dropped them from scikit-image. I think we can skip `musllinux` as pretty niche. If someone needs it we can add it later.

Not sure if you want the `CIBW_TEST_SKIP` here or not. On scikit-image the test suite could not reliably complete within the time limit for aarch64 so we had to skip it here.

---

I agree that `musllinux` is definitely niche, but I would really like to get `aarch64` working if possible. Compiling from source is no fun on a Raspberry Pi!

---

That variable is only to skip the **tests** on aarch64. Definitely agree that we should still build those wheels. I couldn't remember how long this takes for PyFFTW, but for scikit-image it only sometimes would complete within the 6 hour timeout window on GitHub Actions. If tests are completing within the time limit, then let's just not skip them

---

I have removed `musllinux` from the CI builds.

I think the tests run quite quickly, since they are just: `python -c "import pyfftw; import numpy as np; from pyfftw.interfaces import numpy_fft as fft; np.testing.assert_allclose(fft.fft(np.zeros(8)), np.zeros(8));"` (i.e. not the full or even 'quick' test suite).